lightkurve/lightcurve.py

```diff
@@ -1703,6 +1708,120 @@ def errorbar(self, **kwargs):
             ax.set_xlabel("Phase")
         return ax
 
+    def waterfall(self, ax=None, bin_points=1, phmin=-0.5, phmax=0.5, bin_method='mean',
+                        waterfall_method='average', **kwargs):
+        """Plot the folded light curve as a waterfall plot. Keywords are passed
+        to matplotlib.
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes`
+            The matplotlib axes object.
+        bin_points : int
+            How many points should be in each bin
+        phmin : float
+            The minimum phase to plot
+        phmax : float
+            The maximum phase to plot
+        bin_method : str
+            The bin method. Choose from `'mean'` or `'median'`.
+        waterfall_method : 'str'
```

Comment:
Can we just make this `method`?
```suggestion
        method : 'str'
```

Reply:
No because it's easily confused with `bin_method`